test/test_solvers.py

```diff
@@ -44,6 +46,22 @@
 """
 
 
+@pytest.fixture
+def model() -> Model:
+    m = Model()
+
+    x = m.add_variables(4, pd.Series([8, 10]), name="x")
+    y = m.add_variables(0, pd.DataFrame([[1, 2], [3, 4]]), name="y")
+
+    m.add_constraints(x + y, LESS_EQUAL, 10)
+
+    m.add_objective(2 * x + 3 * y)
+
+    m.parameters["param"] = xr.DataArray([1, 2, 3, 4], dims=["x"])
+
+    return m
```

Comment:
Note that this is a duplication of the fixture in `test_io.py`. Importing made the linter unhappy and i was unsure whether this counts as good practice. Happy to switch back, if needed.

Reply:
Ok, @FabianHofmann has used that pattern together with linter ignores previously in `test_common.py` so i'll use it here as well :)